tests/primary.test.mjs

```diff
@@ -225,6 +249,29 @@ describe('primary names', function () {
       });
 
     assertNoResultError(approvePrimaryNameRequestResult);
+    const parsedApproveEvents = parseEventsFromResult(
+      approvePrimaryNameRequestResult,
+    );
+    assert.equal(parsedApproveEvents.length, 1);
+    assert.deepStrictEqual(parsedApproveEvents[0], {
+      _e: 1,
```

Comment:
is `_e` always 1 or does it correspond to length?

Reply:
It's always 1. It's just a flag field for the parser on the CU side.